### DISCUSSION TITLE
OnChange type in customize controller is any

### DISCUSSION THREAD

**Discussion by PeterChen1997** [react-hook-form/react-hook-form#11097] [Q&A]
![image](https://github.com/react-hook-form/react-hook-form/assets/21072589/a36a0452-8ade-4f69-80b8-fb479b52cebb)
https://codesandbox.io/s/agitated-rgb-rsj845

in this file, the onChange input value is any, is there a way to fix it ? Thx~

**Comment by ArtyCodingart** [comment]
This worries me as well.
